src/transformers/models/layoutlm/__init__.py

```diff
@@ -51,6 +51,7 @@
         "LayoutLMForMaskedLM",
         "LayoutLMForSequenceClassification",
         "LayoutLMForTokenClassification",
+        "LayoutLMForQuestionAnswering",
```

Comment:
Should we add them here, given that there might be a new auto mapping for document question answering?

cc @Narsil

Reply:
I think we should definitely add them here, because this part enables you to import `LayoutLMForQuestionAnswering` from `models.layoutlm` and from `transformers`, right? I believe your comment may be about the changes to src/transformers/models/auto/modeling_auto.py

Reply:
This is modeling code, I am not sure I am best to answer your question @NielsRogge .

I am not sure LayoutLM actually implements `ForQuestionAnswering` since it expects some bounding boxes, no ? (But isn't that the same for `ForMaskedLM` and such).

Anyway the PR looks ok, but my biggest worry, is it is actually legit because the inputs seem different (if it's purely optional then it's ok I think). @sgugger maybe ?

Reply:
Hi, hmm not sure why I missed that, I commented at the wrong place :D

Reply:
And @Narsil the model is perfectly legit, see here for a working example: https://huggingface.co/impira/layoutlm-document-qa

Reply:
Closing this as it's been resolved